packages/connectivity/src/scp-cf/destination/destination-from-registration.ts

```diff
@@ -36,7 +36,8 @@ type RegisterDestinationOptions = Pick<
 /**
  * Registers a destination in a cache for later usage.
  *
- * Throws an error if a destination with the same name as the given test destination already exists.
+ * If a destination with the same key is already in the cache it is replaced.
```

Comment:
```suggestion
 * If a destination with the same key is already in the cache, it is replaced.
```

[pp] ...it is overwritten.